guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -4,13 +4,33 @@
 Use this procedure to synchronize templates between your {ProjectServer} and an existing repository.
 
 .Procedure
-. If you want to use HTTPS to connect to the repository and you use a self-signed certificate authentication on your Git server, validate the certificate:
+. If you want to use HTTPS to connect to the repository and you use a self-signed certificate authority (CA) on your Git server:
+.. Create a new directory under the `/usr/share/foreman/` directory to store the Git configuration for the certificate:
 +
 [options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-# sudo -u foreman git config --global http.sslCAPath _Path_To_My_Certificate_
+# mkdir -p /usr/share/foreman/.config/git
 ----
-. If you want to use SSH to connect to the repository, perform the following steps:
+.. Create a file named `config` in the new directory:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# touch /usr/share/foreman/.config/git/config
+----
+.. Allow the `foreman` user access to the `.config` directory:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# chown -R foreman /usr/share/foreman/.config
```

Comment:
I always prefer long options to make them self documenting
```suggestion
# chown --recursive foreman /usr/share/foreman/.config
```

Reply:
Makes sense, let's do it for the whole procedure then.